### PR TITLE
Consider missing loadingExperience in api result

### LINKED ISSUE #5040: PX guide JS error: Cannot read property 'origin_fallback' of undefined
### Current behavior

I'm getting a PX guide JS error: 

> Could not perform page experience check TypeError: Cannot read property 'origin_fallback' of undefined:

Please also provide:
- Page link: https://amp.dev/page-experience/

image

### Expected behavior/code

No JS error.

### Reproduction procedure
Please provide the steps to reproduce the problem:

1. Provide the URL to an AMP page. (You may need to provide a URL that has not been checked recently.)
2. Check it.

- What browser are you using? Chrome
- What O/S are you using? Mac

### PR DESCRIPTION
**Pull request by bezoerb**
fixes #5040

### CODE REVIEW

File: pixi/src/checks/PageExperienceCheck.js [hunk 1]
```diff
@@ -109,10 +109,11 @@ export default class PageExperienceCheck {
   }
 
   createReportData(apiResult) {
-    const fieldMetrics = apiResult.loadingExperience.origin_fallback
-      ? null
-      : apiResult.loadingExperience.metrics;
-    const audits = apiResult.lighthouseResult.audits;
+    const {loadingExperience, lighthouseResult} = apiResult || {};
+    const {origin_fallback: originFallback, metrics} = loadingExperience || {};
```
Comment (sebastianbenz) [inline pixi/src/checks/PageExperienceCheck.js:113]: This looks a bit weird. Seems like `origin_fallback` is never used? What's the real problem here that you're trying to fix?

Comment (bezoerb) [reply pixi/src/checks/PageExperienceCheck.js:113]: `origin_fallback` was already in use. See 012bd02e48ed1d3a109328b44ff3452062ffe858
It's still in use on line 114 ;)

The previous code (`const fieldMetrics = apiResult.loadingExperience.origin_fallback`) failed when there's no `loadingExperience` field available in the api result as described in #5040 See: https://www.googleapis.com/pagespeedonline/v5/runPagespeed?strategy=mobile&url=https://medinathoughts.com/

Comment (bezoerb) [reply pixi/src/checks/PageExperienceCheck.js:113]: I just needed to rename `origin_fallback` to `originFallback` to match the eslint config

Comment (sebastianbenz) [reply pixi/src/checks/PageExperienceCheck.js:113]: My bad. Got confused by the destructuring operator.

**Review by sebastianbenz — APPROVED** [review]

**Pull request merged**